--- erlang-lfe-0.9.2.orig/bin/lfe
+++ erlang-lfe-0.9.2/bin/lfe
@@ -16,25 +16,6 @@
 # Follow symlinks to actual program and set env variables.  We have to
 # cd to each directory following symlinks to handle relative symlinks.
 
-follow_symlink () {
-    cd "$(dirname "$1")" > /dev/null
-    filename="$(basename "$1")"
-    if [ -h "$filename" ]; then
-	follow_symlink "$(readlink "$filename")"
-    else
-	echo "`pwd -P`/$filename"
-    fi
-}
-
-SELF=$(follow_symlink "$0")
-LFE_PROGNAME=`echo $0 | sed 's/.*\///'` # Basically basename
-LFE_BINDIR=$(dirname "$SELF")
-LFE_ROOTDIR=$(dirname "$LFE_BINDIR")
-
-export LFE_ROOTDIR
-export LFE_BINDIR
-export LFE_PROGNAME
-
 i="$#"				# Counter
 e=""				# Eval flag/value
 
@@ -88,4 +69,4 @@ while [ $i -gt 0 ]; do
 done
 
 # Note that ERL_LIBS will find *either* an ebin subdir *or* lib subdir
-ERL_LIBS="$HOME"/.lfe/lib:"$LFE_ROOTDIR":"$LFE_ROOTDIR"/lib:"$ERL_LIBS" exec erl "$@"
+ERL_LIBS="$HOME"/.lfe/lib:"$ERL_LIBS" exec erl "$@"
