Index: pkg-erlang-lfe/bin/lfe
===================================================================
--- pkg-erlang-lfe.orig/bin/lfe
+++ pkg-erlang-lfe/bin/lfe
@@ -16,16 +16,6 @@
 # Follow symlinks to actual program and set env variables.  We have to
 # cd to each directory following symlinks to handle relative symlinks.
 
-follow_symlink () {
-    cd "$(dirname "$1")" > /dev/null
-    filename="$(basename "$1")"
-    if [ -h "$filename" ]; then
-        follow_symlink "$(readlink "$filename")"
-    else
-        echo "$(pwd -P)/$filename"
-    fi
-}
-
 show_help () {
     echo "Usage: `basename $0` [flags] file [args]
 
@@ -36,15 +26,6 @@ show_help () {
                              used by the Erlang VM" >&2
 }
 
-SELF=$(follow_symlink "$0")
-LFE_PROGNAME=$(echo "$0" | sed 's/.*\///') # Basically basename
-LFE_BINDIR=$(dirname "$SELF")
-LFE_ROOTDIR=$(dirname "$LFE_BINDIR")
-
-export LFE_ROOTDIR
-export LFE_BINDIR
-export LFE_PROGNAME
-
 i="$#"                          # Counter
 e=""                            # Eval flag/value
 
@@ -130,6 +111,6 @@ PROJ_LIBS=$(find_libs "./deps")
 # The following works for rebar3
 R3_PROJ_LIBS=$(find_libs "./_build/default/deps"):$(find_libs "./_build/default/lib")
 LFE_HOME_LIBS=$(find_libs "$HOME"/.lfe/lib)
-ALL_LIBS="$LFE_ROOTDIR":"$ERL_LIBS":"$PROJ_LIBS""$R3_PROJ_LIBS""$LFE_HOME_LIBS"
+ALL_LIBS="$ERL_LIBS":"$PROJ_LIBS""$R3_PROJ_LIBS""$LFE_HOME_LIBS"
 ERL_LIBS="$ALL_LIBS" exec erl "$@"

Index: pkg-erlang-lfe/bin/lfescript
===================================================================
--- pkg-erlang-lfe.orig/bin/lfescript
+++ pkg-erlang-lfe/bin/lfescript
@@ -18,16 +18,6 @@
 # Follow symlinks to actual program and set env variables.  We have to
 # cd to each directory following symlinks to handle relative symlinks.
 
-follow_symlink () {
-    cd "$(dirname "$1")" > /dev/null
-    filename="$(basename "$1")"
-    if [ -h "$filename" ]; then
-        follow_symlink "$(readlink "$filename")"
-    else
-        echo "$(pwd -P)/$filename"
-    fi
-}
-
 show_help () {
     echo "Usage: `basename $0` [flags] file [args]
 
@@ -35,15 +25,6 @@ show_help () {
     -flag             Passes configuration flags to be used on startup" >&2
 }
 
-SELF=$(follow_symlink "$0")
-LFE_PROGNAME=$(echo "$0" | sed 's/.*\///') # Basically basename
-LFE_BINDIR=$(dirname "$SELF")
-LFE_ROOTDIR=$(dirname "$LFE_BINDIR")
-
-export LFE_ROOTDIR
-export LFE_BINDIR
-export LFE_PROGNAME
-
 # Set which emulator to use, the default is erl.
 emulator=${LFESCRIPT_EMULATOR-"erl"}
 
@@ -123,5 +104,5 @@ PROJ_LIBS=$(find_libs "./deps")
 # The following works for rebar3
 R3_PROJ_LIBS=$(find_libs "./_build/default/deps"):$(find_libs "./_build/default/lib")
 LFE_HOME_LIBS=$(find_libs "$HOME"/.lfe/lib)
-ALL_LIBS="$LFE_ROOTDIR":"$ERL_LIBS":"$PROJ_LIBS""$R3_PROJ_LIBS""$LFE_HOME_LIBS"
+ALL_LIBS="$ERL_LIBS":"$PROJ_LIBS""$R3_PROJ_LIBS""$LFE_HOME_LIBS"
 ERL_LIBS="$ALL_LIBS" exec "$emulator" +B -boot start_clean "$shebangs" "-noshell" "-run" "lfescript" "start" "$@"
Index: pkg-erlang-lfe/bin/lfe-test
===================================================================
--- pkg-erlang-lfe.orig/bin/lfe-test
+++ pkg-erlang-lfe/bin/lfe-test
@@ -15,21 +15,4 @@
 
 # Follow symlinks to actual program and set env variables.
 
-follow_symlink () {
-    if [ -h "$1" ]; then
-	follow_symlink $(readlink "$1")
-    else
-	echo "$1"
-    fi
-}
-
-SELF=$(follow_symlink "$0")
-LFE_PROGNAME=`echo $0 | sed 's/.*\///'`
-LFE_BINDIR=$(dirname "$SELF")
-LFE_ROOTDIR=$(dirname "$LFE_BINDIR")
-
-export LFE_ROOTDIR
-export LFE_BINDIR
-export LFE_PROGNAME
-
-exec $LFE_BINDIR/lfeexec "$@"
+exec lfeexec "$@"
